sandbox/public/index.html

```diff
@@ -41,13 +41,13 @@
 
     <script>
       alloy("configure", {
-        // edgeDomain: "edgegateway.azurewebsites.net",
-        // edgeDomain:
-        //   "konductor-qe.apps-exp-edge-npe-va6.experience-edge.adobeinternal.net",
+        // edgeDomain: "ENTER YOUR COLLECTION CNAME IF AVAILABLE",
         // optInEnabled: true,
+        // urlActivationEnabled: true,
```

Comment:
Now I'm thinking those should be:

`urlDestinationsEnabled` & `cookieDestinationsEnabled`.

Reply:
I agree.